school/src/main/java/com/hogwarts/school/controller/StudentController.java

```diff
@@ -42,7 +43,7 @@ public ResponseEntity<Student> updateStudent(@RequestBody Student student) {
     }
 
     @DeleteMapping("/removeStudent/{studentId}")
```

Comment:
```suggestion
    @DeleteMapping("/{studentId}")
```
не нужно ничего писать в пути - это ошибка REST